src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -42,6 +50,65 @@ the MIT License. See LICENSE in the project root for license information. -->
+                                    <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="{x:Bind Color, Converter={StaticResource ColorLightenConverter}, Mode=OneWay}"/>
+                                </ResourceDictionary>
+                                <ResourceDictionary x:Key="Dark">
+                                    <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="{x:Bind Color, Converter={StaticResource ColorLightenConverter}, Mode=OneWay}"/>
+                                </ResourceDictionary>
+                                <!-- No High contrast dictionary, let's just leave that unchanged. -->
+                            </ResourceDictionary.ThemeDictionaries>
+                        </ResourceDictionary>
+                    </Button.Resources>
+
+                    <Button.Flyout>
+                        <Flyout>
+                            <ColorPicker Tag="{x:Bind Index, Mode=OneWay}"
+                                         Color="{x:Bind Color, Mode=OneWay}"
+                                         ColorChanged="ColorPickerChanged"/>
+                        </Flyout>
+                    </Button.Flyout>
+                </Button>
+            </DataTemplate>
+
+            <DataTemplate x:Key="ColorTemplate" x:DataType="Color">
+                <Button x:Name="ColorButton"
+                        Background="{Binding Color, ElementName=ColorPicker, Converter={StaticResource ColorToBrushConverter}, Mode=OneWay}"
+                        Style="{StaticResource ColorButtonStyle}">
+                    <Button.Resources>
+                        <!-- Resources to colorize hover/pressed states based on the color of the button -->
+                        <ResourceDictionary>
+                            <ResourceDictionary.ThemeDictionaries>
+                                <ResourceDictionary x:Key="Light">
+                                    <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="{Binding Color, ElementName=ColorPicker, Converter={StaticResource ColorLightenConverter}, Mode=OneWay}"/>
+                                </ResourceDictionary>
+                                <ResourceDictionary x:Key="Dark">
+                                    <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="{Binding Color, ElementName=ColorPicker, Converter={StaticResource ColorLightenConverter}, Mode=OneWay}"/>
+                                </ResourceDictionary>
+                                <!-- No High contrast dictionary, let's just leave that unchanged. -->
+                            </ResourceDictionary.ThemeDictionaries>
+                        </ResourceDictionary>
+                    </Button.Resources>
+
+                    <Button.Flyout>
+                        <Flyout>
+                            <ColorPicker x:Name="ColorPicker"
+                                         Color="{x:Bind Mode=OneWay}"

```

Comment:
Wait shouldn't this one be `TwoWay`? And _not_ have `ColorChanged`?

Reply:
Oh wow. Great catch. Unfortunately, the change wasn't that simple. So I had to create a reference to each of these colors directly.

At this point, we've basically entirely created a view model for a color scheme. #9207 can build off of this and rip some of that out into its own runtime class.

Reply:
Oh, this also let me delete the second data template, which got rid of that weird accessibility behavior.